Apache-Beam-Jobs.md

```diff
@@ -227,6 +228,82 @@ error_pcoll = (
 
 `Runner`s provide the `run()` method used to visit every node (`PValue`) in the pipeline's DAG by executing the edges (`PTransform`s) to compute their values.  At Oppia, we use `DataflowRunner` to have our `Pipeline`s run on the [Google Cloud Dataflow service](https://cloud.google.com/dataflow).
 
+## Writing Apache Beam Jobs
```

Comment:
I feel like this section needs more setup so that a contributor who is new to beam jobs doesn't get lost

Reply:
Done.